config/main.py

```diff
@@ -86,9 +88,20 @@
 asic_type = None
 
 #
-# Breakout Mode Helper functions
+# Helper functions
 #
 
+# Sort nested dict
+def sortDict(data):
```

Comment:
Python function names should be `snake_case`. I do realize that the function below (`readJsonFile`) isn't following this convention either, but it should for consistency.